packages/docusaurus/src/client/App.tsx

```diff
@@ -5,8 +5,11 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+// Load this at the very top!
+// client modules have to be loaded before anything else...
+import './client-lifecycles-dispatcher';
```

Comment:
I actually have no idea we import `client-lifecycles-dispatcher` instead of `@generated/clientModules` here. But side-effects are too hard to reason so I didn't refactor it